### DISCUSSION TITLE
Caso d'uso Trascr_020: gestione provvedimento da trascrivere

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#944] [Supporto servizi cooperativi] [answered]
Per il caso d'uso in oggetto ("_Trascrizione di nascita: Trascrizione di provvedimento straniero adozione di minorenne o maggiorenne in casi particolari richiesta da un privato_"), l'attuale modellazione prevede la presenza obbligatoria sia della sezione:

![image](https://github.com/italia/ansc/assets/26415430/4efdbd94-7885-469e-9736-1fcd97dd5fed)

sia della sezione:

![image](https://github.com/italia/ansc/assets/26415430/da462378-5926-47e2-9d66-b3211a6848b3)

Si intende quindi che il caso d'uso faccia riferimento esclusivamente al caso in cui il privato cittadino richieda all'usc la trascrizione di un provvedimento che un tribunale italiano abbia emesso per rendere efficace in Italia un provvedimento di autorità estera?

**Comment by asqueglia** [comment]
Buongiorno, abbiamo sottoposto il quesito ad ANUSCA. Di seguito la risposta

ANUSCA conferma che i casi particolari si riferiscono solo ai minorenni. Dovrebbe essere l’interessato che chiede la trascrizione della sentenza del tribunale italiano;  se proviene da consolato non è obbligatoria la sentenza del tribunale italiano. 
Quindi procederemo a rendere opzionale la sezione.

**Comment by asqueglia** [comment]
Buonasera con la versione 1.32 è stata corretta la segnalazione. Chiudiamo pertanto la issue

**Comment by antonioluberto** [comment]
Ancora qualche chiarimento necessario:
il caso d'uso in oggetto è:
_Trascr_020 - Trascrizione di provvedimento straniero adozione di minorenne o maggiorenne in casi particolari richiesta da un privato_
quindi relativo al solo caso in cui sia il soggetto privato a chiedere la trascrizione del provvedimento di adozione.

Nella risposta del 2 luglio si indica:
[...]
_Dovrebbe essere l’interessato che chiede la trascrizione della sentenza del tribunale italiano_
[...].

Con il rilascio 1.32: 
- è stata resa opzionale la sezione 
"Sentenza Tribunale Italiano"
- è stata mantenuta l'obbligatorietà della sezione 
"Atto da trascrivere	"

Con il venir meno dell'obbligatorietà della sezione 
"Sentenza Tribunale Italiano"
è ora certamente possibile gestire il caso in cui la richiesta di trascrizione del provvedimento venga da Consolato ("Trascr_021").
E' altresì possibile a questo punto creare - con "Trascr_020" - atto nel quale il soggetto privato richieda direttamente all'usc la trascrizione di un provvedimento emesso da autorità straniera; è effettivamente corretto che un soggetto privato possa richiedere direttamente all'usc la trascrizione di un provvedimento di adozione emesso da autorità estera senza che alcun Tribunale italiano si sia pronunciato per rendere efficace in Italia il provvedimento straniero medesimo?

**Comment by antonioluberto** [comment]
La domanda, alla quale attendiamo ancora risposta, è semplicemente:  ANSC prevede che un soggetto privato possa richiedere direttamente all'usc la trascrizione di un provvedimento di adozione emesso da autorità estera senza che alcun Tribunale italiano si sia pronunciato per rendere efficace in Italia il provvedimento straniero medesimo?

**Comment by asqueglia** [comment]
Buongiorno @antonioluberto, il caso d’uso prevede la sezione Sentenza Tribunale Italiano sebbene come sezione facoltativa; chiederemo ad ANSUCA se rendere obbligatoria la sezione; ribadiamo comunque che è responsabilità dell’USC utilizzare il sistema nei termini consentiti dalla legge, e che non può essere il sistema ad esercitare tutti i controlli bloccanti possibili ed immaginabili per evitare un utilizzo improprio.

**Comment by asqueglia** [comment, marked as answer]
Buongiorno @antonioluberto  da un ulteriore approfondimento con ANUSCA in data odierna, si conferma che è corretto che la sentenza di tribunale italiano sia opzionale ed è corretto che un privato possa richiedere una trascrizione senza che si sia pronunciato un tribunale. Chiudiamo pertanto la discussion
